Example/NYTPhotoViewer-Swift/ViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ViewController.swift
+//  NYTPhotoViewer-Swift
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, NYTPhotosViewControllerDelegate {
+
+    @IBOutlet weak var imageButton : UIButton?
+    private let photos = PhotosProvider().photos
+    private var photosViewController: NYTPhotosViewController
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
+        photosViewController = NYTPhotosViewController(photos: photos as [AnyObject]!)
+        
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+
+        photosViewController.delegate = self
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        photosViewController = NYTPhotosViewController(photos: photos as [AnyObject]!)
+        
+        super.init(coder: aDecoder)
+
+        photosViewController.delegate = self
+    }
+        
+    @IBAction func buttonTapped(sender: UIButton) {
+        photosViewController.delegate = self
+        presentViewController(photosViewController, animated: true, completion: nil)
+        
+        updateImagesOnPhotosViewController(photosViewController, afterDelayWithPhotos: photos)
+    }
+    
+    func updateImagesOnPhotosViewController(photosViewController: NYTPhotosViewController, afterDelayWithPhotos: [ExamplePhoto]) {
+        
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, 5 * Int64(NSEC_PER_SEC))
+        
+        dispatch_after(delayTime, dispatch_get_main_queue()) {
+            for photo in self.photos {
+                if photo.image == nil {
```

Comment:
Current Obj-C style is to not explicitly check for nil, and I think that might make some sense to carry over here.